drivers/phoenixcontact_modbus.c

```diff
@@ -24,17 +24,22 @@
 #include <modbus.h>
 
 #define DRIVER_NAME	"NUT PhoenixContact Modbus driver"
-#define DRIVER_VERSION	"0.04"
+#define DRIVER_VERSION	"0.05"
 
 #define CHECK_BIT(var,pos) ((var) & (1<<(pos)))
 #define MODBUS_SLAVE_ID 192
 
+#define OLD_UPS 0
+#define NEW_UPS 1
```

Comment:
I wonder how soon "old" vs. "new" would become obsolete? Maybe more symbolic names would be better (just a thought)?

Reply:
You're right, i've put the models. To be honest there would be another model (TRIO) but i don't have the hardware to test the communication. As soon as it will arrive i can test the driver and see if changes are needed.

Reply:
Another options can be an `enum` to also constrain the values accepted at compile-time (and get warnings like that a `switch` does not look at all newly defined options).